c-c++.md

```diff
@@ -4,19 +4,32 @@
 To build code with the optimal processor features use the following. If you want to support both Graviton
 and Graviton2 you'll have to limit yourself to the Graviton features.
 
-CPU      | GCC                  | LLVM
----------|----------------------|-------------
-Graviton | `-march=armv8-a+crc+crypto` | `-march=armv8-a+crc+crypto`
+CPU       | GCC                         | LLVM
+----------|-----------------------------|-------------
+Graviton  | `-march=armv8-a+crc+crypto` | `-march=armv8-a+crc+crypto`
 Graviton2 | `-march=armv8.2-a+fp16+rcpc+dotprod+crypto` |`-march=armv8.2-a+fp16+rcpc+dotprod+crypto`
+Graviton3 | `-march=armv8.2-a+fp16+rcpc+dotprod+crypto+sve+bf16+pauth+rng` |`-march=armv8.2-a+fp16+rcpc+dotprod+crypto+sve+bf16+pauth+rand`
 
-Note: GCC-7 does not support `+rcpc+dotprod`.
+Notes:
+- GCC-7 does not support `+rcpc+dotprod`
+- `+sve` is available in gcc >= 8 and in clang >= 6
```

Comment:
i think all of this just makes it more complicated and as i mentioned, it's not going to be emitted by the compiler anyway.